src/components/drawing/index.js

```diff
@@ -887,6 +887,9 @@ drawing.setTextPointsScale = function(selection, xScale, yScale) {
         var transforms;
         var el = d3.select(this);
         var text = el.select('text');
+
+        if(!text.node()) return;
```

Comment:
Caught another bug. `dragmode: 'pan'` is broken on http://rickyreusser.com/plotly-mock-viewer/#text_chart_arrays 

This fixes it:

![peek 2017-10-04 18-24](https://user-images.githubusercontent.com/6675409/31202600-435e410e-a931-11e7-8ce4-faca3e419aaf.gif)

Reply:
ha, good catch! 🎉